app/services/profile_fields/import_from_csv.rb

```diff
@@ -0,0 +1,11 @@
+module ProfileFields
+  class ImportFromCsv
+    HEADERS = %i[label input_type placeholder_text description].freeze
+
+    def self.call(file)
+      CSV.foreach(file, headers: HEADERS, skip_blanks: true) do |row|
```

Comment:
We can tweak the CSV options if needed. I generally prefer semicolons or tabs as separators, since not all programs implement quoting correctly (see [RFC4180](https://tools.ietf.org/html/rfc4180,) sections 2.4 and 2.5), but I left it at comma for now.